app/web/template/layout/page.gohtml

```diff
@@ -103,11 +79,28 @@
               beta
             </strong>
             <span class="govuk-phase-banner__text">
-              This is a new service – your <a class="govuk-link" href="#">feedback</a> will help us to improve it.
+              {{ .L.HTML "newServiceContent" }}
             </span>
           </p>
         </div>
-
+        
+        <div class="govuk-clearfix">          
+          <ul class="trans-switch" id="language-switch" aria-label="Change your language">
+            <li class="govuk-list trans-switch__list govuk-body-s" id="language-switch-en" lang="en-gb">
+              {{ if isWelsh .Lang }}<a href="/" class="govuk-link"><span class="govuk-visually-hidden">Change language to </span>{{ end }}
+                {{ if isEnglish .Lang }}<span class="govuk-visually-hidden">Current language is </span>{{ end }}
+                English
+                {{ if isWelsh .Lang }}</a>{{ end }}
```

Comment:
Are there limitations with using the translation functions with this markup too?

Reply:
I guess not, although it would look weird because the words would be the same in both `en.json` and `cy.json`

Reply:
Ah gotcha - makes sense 👍